src/library.js

```diff
@@ -4052,8 +4052,19 @@ LibraryManager.library = {
     return 0; // we cannot succeed
   },
 
+  _Unwind_RaiseException__deps: ['__cxa_find_matching_catch', '$EXCEPTIONS'],
   _Unwind_RaiseException: function(ex) {
-    abort('Unwind_RaiseException');
```

Comment:
Probably yes, but `cxa_throw` takes three parameters and I'm not sure how to extract the relevant values from `ex`.